model/project_parser.go

```diff
@@ -438,23 +430,23 @@ func LoadProjectInto(data []byte, identifier string, project *Project) error {
 
 // projectFromYAML reads and evaluates project YAML, returning a project and warnings and
 // errors encountered during parsing or evaluation.
-func projectFromYAML(yml []byte) (*Project, []error) {
-	intermediateProject, errs := createIntermediateProject(yml)
-	if len(errs) > 0 {
-		return nil, errs
+func projectFromYAML(yml []byte) (*Project, error) {
+	intermediateProject, err := createIntermediateProject(yml)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating intermediate project")
```

Comment:
I don't think these 2 errors should be wrapped since they'll be seen by users and won't have any meaning